resources/shaders/dxr_shader_template.hlsl

```diff
@@ -0,0 +1,62 @@
+#ifndef __SHADER_NAME_HLSL__
```

Comment:
The template is great! Might want to put it on our Wiki?

Reply:
Will go there